webviz_examples/webviz-full-demo.yml

```
@@ -280,9 +280,7 @@ layout:
                   ensembles: *hm_ensembles
                   obsfile: *obsfile
                   options:
-                    vector1: WWCT:A1
-                    vector2: WGOR:A1
-                    vector3: WBHP:A1
+                    vectors: [WWCT:A1, WGOR:A1, WBHP:A1]
```

```suggestion
                    vectors:
                      - WWCT:A1
                      - WGOR:A1
                      - WBHP:A1
```
is perhaps somewhat more known to the non-programmers among th eusers.